source/pdo_sqlsrv/pdo_stmt.cpp

```diff
@@ -782,14 +782,11 @@ int pdo_sqlsrv_stmt_get_col_data( _Inout_ pdo_stmt_t *stmt, _In_ int colno,
 
         // set the encoding if the user specified one via bindColumn, otherwise use the statement's encoding
         // save the php type for next use
-        if (driver_stmt->current_meta_data[colno]->sqlsrv_php_type.typeinfo.type != SQLSRV_PHPTYPE_INVALID) {
-            sqlsrv_php_type = driver_stmt->current_meta_data[colno]->sqlsrv_php_type;
-        }
```

Comment:
Did this cause problems with the new tests?

Reply:
Yes, as shown in the pdo test, the user may choose different encoding or type for every single fetch